kubectl/kubectl.go

```diff
@@ -112,6 +112,12 @@ func (k *Kubectl) NamespaceWithReadyPod(namespace string, labelName string) (boo
 			if !s.Ready {
 				return false, nil
 			}
+			if !s.Started {
```

Comment:
Might sounds a duplication to test `Started` and `Ready` but I think it's safer to check both of them to try to avoid false positive.